Document Raven studio access for RavenDB 5 (#5899)
Particular/docs.particular.net: servicecontrol/audit-instances/creating-config-file.md

## Troubleshooting

ServiceControl Audit stores its data in a RavenDB embedded instance. If, during troubleshooting, direct access to the RavenDB instance is required while ServiceControl Audit is running use the following instructions.

NOTE: [Maintenance mode](/servicecontrol/audit-instances/maintenance-mode.md) is the recommended way to review documents in the embedded RavenDB instance.

WARNING: The ServiceControl RavenDB embedded instance is used exclusively by ServiceControl Audit and is not intended for external manipulation or modifications.

### RavenDB 5

> If on Version 4.26 and using [the new RavenDB 5 persister](/servicecontrol/upgrades/new-persistence.md) browse to:

Reviewer: ```suggestion
For instances running version 4.26 and above, which are configured to use [the new RavenDB 5 persistence](/servicecontrol/upgrades/new-persistence.md) browse to:
```

```no-highlight
http://localhost:{configured ServiceControl instance maintenance port}
```

to access the internal database via [the RavenDB studio interface](https://ravendb.net/docs/article-page/5.4/csharp/studio/overview).

### RavenDB 3.5

> If on Version 4.25 and below or using the old RavenDB 3.5 persister the ServiceControl Audit instance can be configured to expose the RavenDB studio.

Reviewer: ```suggestion
For instances running version 4.25 and below or using the old RavenDB 3.5 persistence the ServiceControl Audit instance can be configured to expose the RavenDB studio.
```

## Troubleshooting

ServiceControl Audit stores its data in a RavenDB embedded instance. If, during troubleshooting, direct access to the RavenDB instance is required while ServiceControl Audit is running use the following instructions.

> ### Version 4.26 and above

Reviewer 2: Do we want to mention here, ner persistance? As you can have the old instance upgraded to the 4.26

Author: Good catch, I have reworked the sections to convey this now

Browse: 

```no-highlight
http://localhost:{configured ServiceControl instance maintenance port}
```

> to access the internal database via [the RavenDB studio interface](https://ravendb.net/docs/article-page/5.4/csharp/studio/overview).

Author: Do we want to mention maintenance mode here as well?
